docs/core/domains.rst

```diff
@@ -29,7 +29,21 @@ As an example, the ``DefaultDomain`` has the following yaml definition:
 **What does this mean?**
```

Comment:
Can we instead of "the `DefaultDomain`" (this... isn't a thing) just say "the domain created by `rasa init`"

Reply:
haha I didn't write that but yeah that makes more sense